dockerfiles/Dockerfile.packager

```
@@ -0,0 +1,7 @@
+FROM node:lts-alpine
+
+RUN apk add --update bash
+
+WORKDIR /app
+
+CMD ["/bin/bash", "features/scripts/create_package.sh"]
```

Assuming this is executable, it doesn't need to be run with bash/sh specifically

```suggestion
CMD "features/scripts/create_package.sh"
```